kafka_utils/util/metadata.py

```diff
@@ -43,6 +43,14 @@ def get_topic_partition_metadata(hosts):
     return topic_partitions
 
 
+def get_unavailable_brokers(zk, partition_metadata, unavailable_brokers):
```

Comment:
I think this would be simpler if "get_unavailable_brokers" does not take the unavailable_brokers parameter.

It could be something like this:

```
unavailable_brokers.update(
    get_unavailable_brokers(zk, partition_metadata)
)
```

Reply:
Or equivalent:

```
unavailable_brokers |= get_unavailable_brokers(zk, partition_metadata)
```